lib/zold/version.rb

```diff
@@ -23,5 +23,6 @@
 # Copyright:: Copyright (c) 2018 Yegor Bugayenko
 # License:: MIT
 module Zold
-  VERSION = '0.0.0'.freeze
+  VERSION  = '0.0.0'.freeze
+  PROTOCOL = '0.0.1'.freeze
```

Comment:
@botanicus this is not correct. According to our White Paper, protocol is just a simple Integer. Current protocol number is `1` in all wallets.